aws/resource_aws_subnet.go

```diff
@@ -320,7 +320,7 @@ func resourceAwsSubnetDelete(d *schema.ResourceData, meta interface{}) error {
 	wait := resource.StateChangeConf{
 		Pending:    []string{"pending"},
 		Target:     []string{"destroyed"},
-		Timeout:    10 * time.Minute,
+		Timeout:    20 * time.Minute,
```

Comment:
Should we update this to use `d.Timeout(schema.TimeoutDelete)`? 🤔 I'm a little wary of increasing the `DependencyViolation` timeout for all operators when it really should be more customizable right here.

Reply:
If matching the overall timeout is the new idiom then I'm happy to change this to match?